test/index.html

```diff
@@ -12,6 +12,7 @@
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
     <script src="../../web-component-tester/browser.js"></script>
+    <script src="../../test-fixture/test-fixture-mocha.js"></script>
```

Comment:
Ugh, wrong comment order. https://github.com/PolymerElements/seed-element/pull/85#discussion_r46194450 should be first :(